pyalgotrading/algobulls/api.py

```diff
@@ -193,30 +197,25 @@ def get_strategy_details(self, strategy_code: str) -> dict:
 
         Return:
             JSON
-
-        Info: ENDPOINT
-            `GET` v2/user/strategy/build/python
         """
-        params = {'strategyCode': strategy_code}
-        endpoint = f'v2/user/strategy/build/python'
+        params = {}
+        endpoint = f'v3/build/python/user/strategy/code/{strategy_code}'
+        print(endpoint)
         response = self._send_request(endpoint=endpoint, params=params)
         return response
 
-    def search_instrument(self, instrument: str) -> dict:
+    def search_instrument(self, tradingsymbol: str, exchange: str) -> dict:
         """
 
         Args:
-            instrument: instrument key
+            tradingsymbol: instrument tradingsymbol
+            exchange: instrument exchange
 
         Returns:
             JSON Response
-
-
-        Info: ENDPOINT
```

Comment:
some methods have

`Info: ENDPOINT`

and some dont

can we standardize this and make it consistent everywhere?